Added yaml file for OrderForm

spartanbear opened the pull request:
Added yaml file for OrderForm for both Dev and Test

bjss-sean-rose committed:
Added yaml file for OrderForm

Added yaml file for OrderForm for both Dev and Test

stuart-bass-cgi commented on Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml:
```diff
@@ -0,0 +1,81 @@
+          value: "development"
+          ##The base uri the app is running on to pass to identity service for redirection
+        - name: APP_BASE_URI
+          value: "https://buyingcataloguetestpublic.uksouth.cloudapp.azure.com"
```
shouldn't this be dev?

stuart-bass-cgi commented on Kubernetes/OrderForm/test-orderform-kubernetes-manifest.yml:
```diff
@@ -0,0 +1,81 @@
+          value: "https://buyingcataloguetestpublic.uksouth.cloudapp.azure.com"
+          ##Environment
+        - name: NODE_ENV
+          value: "testelopment"
```
I think this variable should be removed

stuart-bass-cgi commented on Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml:
```diff
@@ -0,0 +1,81 @@
+        env:
+          ##Base URL
+        - name: BASE_URL
+          value: "orderform"
```
I think this should be /order

bjss-sean-rose committed:
Added probes and additional variables

Added readiness and liveness probes, plus additional env variables. Comments have been amended

stuart-bass-cgi commented on Kubernetes/OrderForm/dev-orderform-kubernetes-manifest.yml:
```diff
@@ -0,0 +1,103 @@
+          limits:
+            cpu: 500m
+        livenessProbe:
+          httpGet:
```
order form doesn't have live / ready yet

spartanbear committed:
Removing probes from dev

spartanbear committed:
Removed probes from test
